aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -274,6 +274,18 @@ Tensor empty_like(
 
   auto memory_format = options.memory_format_opt().value_or(MemoryFormat::Preserve);
 
+  if (options.layout() == kSparseCsr && self.is_sparse_csr()) {
```

Comment:
Is this change used in this PR now?

Reply:
Nope, I've removed it from this PR. :) This can instead go in `torch.randn_like` PR. Apologies should have noticed before. :)